server/data/restClient.test.ts

```diff
@@ -56,23 +56,6 @@ describe('POST', () => {
     })
   })
 
-  it('Should not retry by default', async () => {
```

Comment:
Do you want `POST` requests to be retried?

Reply:
hasn't been doing retries currently so I am not sure if any current implementation will be affected.

Reply:
I have updated the test to reintroduce this method. retries are not currently used but could be used if needed at a later date as it now uses the new restClient.